src/snowcli/cli/project/definition.py

```diff
@@ -78,3 +78,21 @@ def generate_local_override_yml(project: Union[Dict, YAML]) -> YAML:
         }
 
     return as_document(local, project_override_schema)
+
+
+def default_app_package(*args):
+    project_name: str = args[0]
+    user = clean_identifier(get_env_username() or DEFAULT_USERNAME)
+    return f"{project_name}_pkg_{user}"
+
+
+def default_role():
+    conn = snow_cli_global_context_manager.get_connection()
+    return conn.role
+
+
+def default_application(*args):
+    project_name: str = args[0]
```

Comment:
Can this (and `default_app_package` above) just be regular functions that take `(project_name: str)`?

Reply:
My bad, this is left over from a previous commit and I did not revisit them after that - will change in the next commit.